meson_options.txt

```diff
@@ -33,6 +33,8 @@ option('use_sys_openssl', type: 'feature', value: 'disabled')
 option('use_sys_tree_sitter', type: 'feature', value: 'disabled')
 option('use_sys_libuv', type: 'feature', value: 'auto', description: 'Whether to force, suggest or not use at all the system version of libuv. If system version is not found, one is built statically, unless use_libuv is false.')
 option('use_libuv', type: 'boolean', value: true, description: 'If true, libuv is used to handle remote features')
+option('use_swift_demangler', type: 'boolean', value: true, description: 'If false, disables the swift demangler')
+option('use_gpl', type: 'boolean', value: true, description: 'Set to false when you want to disable gpl code')
```

Comment:
I'd add "not completely supported yet" because I think there's other code that is GPL which is not disabled if you set this to false, right?

Reply:
if you mean in rizin codebase, yes, but if it is referred to libdemangler, no. it is supported and not built when this flag is enabled.

Reply:
i mean in rizin codebase. This is rizin meson file, so one may think that it applies to the whole rizin.

Reply:
then no. but i could find all GPL code and add the `#if USE_GPL`.

Reply:
i think it's a separate thing. Maybe let's create an issue and do it separately after this if you want.

Reply:
ok, i have added the comment. i will make another PR with all GPL guards once this is merged.